source/configuration_manual/mail_location/obox/reference_counted_mapping.rst

```diff
@@ -17,3 +17,32 @@ Reference counting requires these additional tables:
 
 Creation of these tables (and the corresponding dictmap settings required) are described in :ref:`simple_mapping`, :ref:`scality_sproxyd` or :ref:`swift_dict_map` pages respectively.
 
+.. _reference_counted_mapping_allow_copy_between_users:
+
+Reference counting with ``obox_allow_copy_between_users``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: v2.3.11
+
+If the setting ``obox_allow_copy_between_users`` is set to ``yes`` the query
+using the ``user_mailbox_objects_reverse`` table for reference counting must be
+modified in order to support correct reference counting per user.
+As mails can be shared between users in storage the query must return the user,
+so the ``value_field`` and ``value_type`` for the map:
+``shared/dictrevmap/$object_id/$object_name`` must be changed to ``u,g`` and
+``string,hexblob``.
```

Comment:
This change can be done even when obox_allow_copy_between_users=no, right? I think this text belongs to the upgrade notes, and the dovecot-dict-cql.conf.ext map should just be changed directly to this new format.